gio/tests/appmonitor.c

```diff
@@ -5,7 +5,6 @@ static gboolean
 create_app (gpointer data)
 {
   const gchar *path = data;
-  gchar *file;
```

Comment:
@cosimoc The upstream commit is slightly different and doesn't work; the appmonitor test still fails. Here's the diff you need to make it work; I suggest reopening that upstream bug and adding this...
```diff
diff --git a/gio/tests/appmonitor.c b/gio/tests/appmonitor.c
index e0068e1..34661d8 100644
--- a/gio/tests/appmonitor.c
+++ b/gio/tests/appmonitor.c
@@ -79,7 +79,7 @@ test_app_monitor (void)
 
   g_timeout_add_seconds (3, quit_loop, loop);
 
-  delete_app (path);
+  delete_app (app_path);
 
   g_main_loop_run (loop);
 
```